src/Elastic.Apm/Helpers/ConfigHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Text;
+using Elastic.Apm.Config;
+using Elastic.Apm.Logging;
+
+namespace Elastic.Apm.Helpers
+{
+	public class ConfigHelper
```

Comment:
As I see, it's called only from `src/Elastic.Apm.AspNetFullFramework/ElasticApmModule.cs`. Maybe it's better to store the class also in `Elastic.Apm.AspNetFullFramework` project. What do you think? It will allow not to add `System.Configuration` dependency which makes sense only for .NET Framework applications.

Reply:
if i push it to Elastic.Apm.AspNetFullFramework it will not reusable by other projects and tests